pep-0646.rst

```diff
@@ -722,6 +722,187 @@ Normal ``TypeVar`` instances can also be used in such aliases:
     # T bound to Any, Ts to an Tuple[Any, ...]
     Foo
 
+
+Substitution in Aliases
+-----------------------
+
+In the previous section, we only discussed simple usage of generic aliases
+in which the type arguments were just simple types. However, a number of
+more exotic constructions are also possible.
+
+
+Type Arguments can be Variadic
+''''''''''''''''''''''''''''''
+
+First, type arguments to generic aliases can be variadic. For example, a
+``TypeVarTuple`` can be used as a type argument:
+
+::
+
+    Ts1 = TypeVar('Ts1')
```

Comment:
Just FYI in case you are unaware—what you have is perfectly valid syntax, but with a bare `::` you can also do

```rst
``TypeVarTuple`` can be used as a type argument::

    Ts1 = TypeVar('Ts1')
```

Reply:
Ohhh, good to know - thanks for the heads up! I'll bear this in mind for the future.